src/plugins/intel_cpu/src/nodes/deconv.cpp

```diff
@@ -774,9 +830,14 @@ void Deconvolution::prepareParams() {
     auto engine = getEngine();
     auto builder = [&engine](const DeconvKey& key) -> executorPtr {
         std::shared_ptr<DnnlDesriptor> desc;
+        dnnl::memory::desc dnnlBiasDesc;
         if (key.isInt8) {
-            desc = createInt8MkldnnDeconvDesc(key.inp0->getDnnlDesc(), key.inp1->getDnnlDesc(), key.out->getDnnlDesc(),
-                                              key.stride, key.dilation, key.paddingL, key.paddingR);
+            if (key.bias)
+                // WA to align IR bias representation (3 to 5 rank tensors) to oneDNN representation (1 rank tensor)
+                dnnlBiasDesc = key.bias->getDnnlDesc().reshape({key.out->getDnnlDesc().dims()[1]});
```

Comment:
I would say it is better to use key.out->getShape().getStaticDims()[1];, this is just faster. `dims()` call creates a new vector,

Reply:
OK, will change